packages/sdk/test/utils/generatePromotionTier.ts

```diff
@@ -0,0 +1,9 @@
+import { generateCampaignWithOnePromotionTier } from './generateCampaignWithOnePromotionTier'
+
+export const generatePromotionTier = async () => {
```

Comment:
I wouldn't say I like this function - it's misleading as it suggests that you can create a promotionTier without a campaign